test/built-ins/Array/prototype/group/callback-arg.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.group
+description: Array.prototype.group calls function with correct arguments
+info: |
+  22.1.3.14 Array.prototype.group ( callbackfn [ , thisArg ] )
+
+  ...
+
+  6. Repeat, while k < len
+
+      a. Let Pk be ! ToString(𝔽(k)).
+      b. Let kValue be ? Get(O, Pk).
+      c. Let propertyKey be ? ToPropertyKey(? Call(callbackfn, thisArg, « kValue, 𝔽(k), O »)).
+      d. Perform ! AddValueToKeyedGroup(groups, propertyKey, kValue).
+      e. Set k to k + 1.
+  ...
+features: [Array.prototype.group]
+---*/
+
+
+const arr = [-0, 0, 1, 2, 3];
+
+const sentinel = {};
+let calls = 0;
```

Comment:
`sentinel` is not used, this seems to be left over from one of the other tests?
```suggestion
let calls = 0;
```